### ISSUE TITLE
Locked controls if disturbed while waking up

### ISSUE THREAD

**Issue by Dehela** [Try/OpenGothic#308]
If someone tries to talk to you while you are getting off the bed, you would end up being not able to move and interact with anything anymore, so only option would be loading a save file. Can be easily reproduced by starting a new game and try to sleep till midnight in Lobart's house, his wife (if you didn't interact with her earlier) would wake you up and soft-lock the game.

Also, while it should be a separate ticket (feels like I'm littering with those), there is sleeping animation issue with NPCs. If one was sleeping and would want to talk to you when you come closer, NPC would hover up a bit, spin 180 in the air with sleeping posture and start talking to you just like that. You can see an example below.

![1](https://user-images.githubusercontent.com/111627003/189986578-46cb8b00-0ec0-4370-9871-947b10e47540.jpg)

**Comment by Dehela** [comment]
Another similar case of soft-lock. Praying for paladins in Monastery quite often results in getting stuck because paladin Sergio would talk to you right after that. Seem like the camera is getting caught in weird position which might cause issues. Among seven of so attempts, only once I somehow avoided getting stuck.

**Comment by Try** [comment]
Hi, @Dehela !
Should work now, not closing issue for now, since my reproduction is not exact. Would be perfect, if you have time to test it ;)

**Comment by Dehela** [comment]
Hi!

I checked all three cases that I reported here, and so far all of them are working correctly now, thanks to pause for animations to end. Thank you!
